gamefixes-steam/2677660.py

```diff
@@ -0,0 +1,9 @@
+"""Game fix for Indiana Jones and the Great Circle"""
+
+from protonfixes import util
+
+
+def main() -> None:
+	"""Bad performance on Nvidia due to poor VRAM utilization"""
+	util.set_environment('DXVK_NVAPI_GPU_ARCH', 'GA100')
```

Comment:
Sorry, but does setting __GL_13ebad=0x1 not suffice for this game? What happens if you do not set DXVK_NVAPI_GPU_ARCH=GA100?

Reply:
According to the OP:

> Please note that the game currently needs the env variable DXVK_NVAPI_GPU_ARCH=GA100 %command% to fake as an Ampere card, in order to avoid a game crash under Proton.